finesse/config.py

```diff
@@ -53,6 +53,25 @@
 EM27_SENSORS_TOPIC = "em27_sensors"
 """The topic name to use for EM27 sensor-related messages."""
 
+DECADES_HOST = "localhost"
+"""The IP address or hostname of the DECADES server."""
+
+DECADES_SENSORS_URL = "http://{host}/decades/livedata"
+"""The URL of the DECADES sensor data endpoint."""
+
+DECADES_SENSORS_POLL_INTERVAL = 60.0
+"""Poll rate for DECADES sensors."""
+
+DECADES_SENSORS_TOPIC = "decades_sensors"
+"""The topic name to use for DECADES sensor-related messages."""
+
+DECADES_SENSORS_QUERY_LIST = [
```

Comment:
Shall we also query the `time` if that is available, so it can be later correlated with other measurements?

Reply:
Yeah, agree. Not sure how best to do this yet. Timestamps are returned from decades with each query anyway:

```
{
    "deiced_true_air_temp_c": [
        20.672006730159012
    ],
    "nondeiced_true_air_temp_c": [
        20.790049884423468
    ],
    "utc_time": [
        1708360706.0
    ]
}
```